### ISSUE TITLE
Quick Start: Uncaught (in promise) TypeError: Icon is undefined

### ISSUE THREAD

**Issue by psusu** [vue-leaflet/vue-leaflet#251]
On a new Install of vue 3 running on VITE 3.2.5 I'm trying to use the Quick Start example but it throws me this error:

**Uncaught (in promise) TypeError: Icon is undefined** on `await resetWebpackIcon(Icon);` in **utils.js:70** and **LMap.vue:247**
can you help me? Thank you!

**Comment by paulvonber** [comment]
Exactly same issue

**Comment by athoma13** [comment]
Same here...

**Comment by pascalfavre** [comment]
I have this same error with a Vite instance

**Comment by pascalfavre** [comment]
Hello, I was able to make it work by simply adding :useGlobalLeaflet="false" to my l-map component.

**Comment by anktx** [comment]
@pascalfavre, yes, everything works, except for the Icon component.

**Comment by mikeu** [comment]
Thanks everyone for reporting this issue. I've reproduced it myself, but it seems to be somewhat intermittent.

Can anyone who's seeing this problem verify whether it still happens in the quick start code for them if they import Leaflet before the vue-leaflet components? I've so far been unable to reproduce this issue locally if I have `import "leaflet"` before `import { LMap, LTileLayer } from "@vue-leaflet/vue-leaflet"` in the sample code.

**Comment by anktx** [comment]
Thank you, @mikeu!
`import "leaflet"` - it works for me.

**Comment by lu-min** [comment]
> Thanks everyone for reporting this issue. I've reproduced it myself, but it seems to be somewhat intermittent.
> 
> Can anyone who's seeing this problem verify whether it still happens in the quick start code for them if they import Leaflet before the vue-leaflet components? I've so far been unable to reproduce this issue locally if I have `import "leaflet"` before `import { LMap, LTileLayer } from "@vue-leaflet/vue-leaflet"` in the sample code.

Yes. `import "leaflet"` before `import { LMap, LTileLayer } from "@vue-leaflet/vue-leaflet"` in the sample code solved my problem. Thank you!

**Comment by alessiostalla** [comment]
> Thanks everyone for reporting this issue. I've reproduced it myself, but it seems to be somewhat intermittent.
> 
> Can anyone who's seeing this problem verify whether it still happens in the quick start code for them if they import Leaflet before the vue-leaflet components? I've so far been unable to reproduce this issue locally if I have `import "leaflet"` before `import { LMap, LTileLayer } from "@vue-leaflet/vue-leaflet"` in the sample code.

Works for me, too.
